### ISSUE TITLE
[3.3.0RC1]  Camera auto upload fails

### ISSUE THREAD

**Issue by tpafe** [nextcloud/android#2972]
### Actual behaviour
with the xiaomi mi5s the automatic loading of the camera folder does not work correctly. Sometimes you take a picture and upload after a few seconds as in previous versions, and now it is random. You can upload to 10 minutes, 30 minutes or not upload directly until you make another photo. as if it did not detect that there are new photos. obviously always with wifi connection and energy savings and self-execution allowed.

### Expected behaviour
- Tell us what should happen
 
### Steps to reproduce
1. take foto
2. 
3. 

### Environment data
Android version: 7.0

Device model: xiaomi mi5s

Stock or customized system: miui 9

Nextcloud app version: 3.3.0 rc1

Nextcloud server version: 13.0,6

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
This is a bit weird as we did not changed anything related to AutoUpload in latest version.

**Comment by rwat1** [comment]
Hi guys, I posted a solution here that works for many users. 

https://github.com/nextcloud/android/issues/2834#issuecomment-413856687

TLDR: clearing NC app cache/data  + reinstalling NC app.

**Comment by rwat1** [comment]
@tobiasKaminsky 

I have theory that every time Android updates or alters it's security settings, the data configuration settings for NC app gets corrupted or de-linked or messed up.

That's just my opinion because clearing NC app cache/data + reinstalling NC app fixes the autoupload problem.
